README.md

```diff
@@ -303,6 +303,56 @@ users, so make sure to understand what you are doing.
 * `doil mail:stop` stops the mail server
 * `doil mail:restart` restarts the mail server
 
+### xdedug
+
+**doil** provides two options to enable xdebug for the given instance.  
+
+First an instance can be created with the '-x' flag, or alternatively 
```

Comment:
"Option 1: An instance can ..."
[...]
"Option 2: You can apply ..."

Reply:
Done!